src/com/google/javascript/jscomp/FindModuleDependencies.java

```diff
@@ -97,6 +98,17 @@ public void visit(NodeTraversal t, Node n, Node parent) {
 
     if (supportsEs6Modules && n.isExport()) {
       moduleType = ModuleType.ES6;
+      if (n.getBooleanProp(Node.EXPORT_DEFAULT)) {
+        // export default
+      } else if (n.getBooleanProp(Node.EXPORT_ALL_FROM)) {
+        // export * from 'moduleIdentifier';
+        // not yet supported by closure ES6 transpilation: 'Wildcard export' is not yet implemented.
+      } else if (n.hasTwoChildren()) {
```

Comment:
This is the only case we care about since I believe the only way an export has more than one child is if it exports from another module. There's no need to special case `export *`. While transpilation may not handle this case yet, we still want to add it to the module graph.

Can you simplify this case to that single if test?